spec/facets/StorageDatasetFacet.json

```diff
@@ -17,12 +17,13 @@
             "fileFormat": {
               "description": "File format with allowed values: parquet, orc, avro, json, csv, text, xml.",
               "type": "string"
+            },
+            "physicalLocation": {
+              "description": "Physical location of the stored dataset.",
+              "type": "string"
             }
           },
-          "additionalProperties": true,
-          "required": [
-            "storageLayer"
```

Comment:
is this a backwards compatible change?

Reply:
Ufortunately, it's not backwards compatible. We'll need a separate facet for that.
Thanks for spotting this.